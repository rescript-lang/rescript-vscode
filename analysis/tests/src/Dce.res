// ^dce

let x = 10
