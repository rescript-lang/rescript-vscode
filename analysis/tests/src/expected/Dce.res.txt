DCE src/Dce.res
issues:2

